mm2src/coins_activation/src/token.rs

```diff
@@ -104,6 +124,7 @@ impl From<BalanceError> for EnableTokenError {
 #[derive(Debug, Deserialize)]
 pub struct EnableTokenRequest<T> {
     ticker: String,
+    protocol: Option<CoinProtocol>,
```

Comment:
I guess we can call this `custom_protocol`?

Reply:
The protocol is not custom though. The token is what is custom.